contracts/SnappAuction.sol

```diff
@@ -120,25 +120,7 @@ contract SnappAuction is SnappBase {
         bytes32 orderHash;
         bytes memory orderData;
         for (uint i = 0; i < numOrders; i++) {
-            orderData = packedOrders.slice(26*i, 26);
-
-            uint8 buyToken = BytesLib.toUint8(orderData, 0);
-            uint8 sellToken = BytesLib.toUint8(orderData, 1);
-
-            uint96 buyAmount;
-            assembly {  // solhint-disable no-inline-assembly
-                buyAmount := mload(add(add(orderData, 0xc), 2))
-            }
-            uint96 sellAmount;
-            assembly {  // solhint-disable no-inline-assembly
-                sellAmount := mload(add(add(orderData, 0xc), 14))
-            }
-            orderHash = sha256(
-                abi.encodePacked(
-                    orderHash,
-                    encodeOrder(accountId, buyToken, sellToken, buyAmount, sellAmount)
-                )
-            );
+            orderhash = calculateNextOrderHashIteration(packedOrders.slice(26*i, 26), orderhash);
```

Comment:
What happened with `bytes memory orderData` from above? this looks like an unused variable now?
Do we know this works if the auction index is increased in the middle of this loop?

Also, I think this function name is way too long! Suggest something like `parseOrder`

Reply:
Suggest `updateOrderHash`